homecoming/apps/scores/migrations/0003_scoreboard_staff_score.py

```diff
@@ -6,13 +6,13 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('scores', '0002_auto_20200715_2206'),
+        ("scores", "0002_auto_20200715_2206"),
```

Comment:
Migrations shouldn't be formatted.

Reply:
I was thinking about this as well - every other migration seems to have been formatted by the past maintainers in the same way, so I decided to do so as well. Additionally, the format script automatically did this, so this would require a change to the black exclusion list, probably across multiple CSL sites.